cast/src/lib.rs

```diff
@@ -78,7 +78,7 @@ pub fn get_account<'a>(
     ));
     let address = FieldElement::from_hex_be(&account_info.address)
         .with_context(|| format!("Failed to convert account address {} to FieldElement", &account_info.private_key))?;
-    let mut account = SingleOwnerAccount::new(provider, signer, address, network.get_chain_id());
+    let account = SingleOwnerAccount::new(provider, signer, address, network.get_chain_id());
```

Comment:
Please also change commands (they expect &mut account)

Reply:
From what I understand it doesn't matter for the outer code, the `mut` was redundant here because the function just returns an object, and whether it is mut or not, it depends on the outer declaration.

Reply:
can we resolve this?

Reply:
let's. but please mark this PR as draft for now, so it won't get merged by accident (we should still discuss changes to interface with Maks when he's back)